app/core/application.coffee

```diff
@@ -78,14 +78,16 @@ Application = {
     $('body').addClass 'picoctf' if window.serverConfig.picoCTF
     if $.browser.msie and parseInt($.browser.version) is 10
       $("html").addClass("ie10")
-    @tracker = new Tracker()
+
+    @tracker = new Tracker(store)
+    window.tracker = @tracker
```

Comment:
Is it valuable to keep the error that checks that a tracker doesn't already exist?

https://github.com/codecombat/codecombat/pull/5776/files#diff-e4e290d1032eadce6a44edd0fa6c7f4aL17

Reply:
That error was necessary because we were writing window.tracker from the constructor of the Tracker, potentially writing it multiple times if we initialized multiple trackers.  Instead of exposing the tracker at the window from the Tracker constructor (not a great side effect to have in a constructor), we just expose it once during application init.  So no error checking necessary!